tox.ini

```diff
@@ -114,10 +131,8 @@ commands =
 skip_install = True
 passenv = FORCE_COLOR
 deps =
-    check_manifest
-    build
-    twine
+    build==1.0.3
+    twine==4.0.2
 commands =
-    check-manifest -v
```

Comment:
This might just be for my own edification....but I was under the impression that this ensures the sdist matches what's in git. Is this enforced by default now? Or no longer a concern? Just not sure I fully understand this transition...

Reply:
It's enforced by way of setuptools_scm. It controls the list of files included in the sdist to match everything that is in source control. Essentially we get a dynamic MANIFEST.in for free.